libs-haskell/da-hs-base/src/DA/Service/Logger/Impl/GCP.hs

```diff
@@ -311,6 +310,9 @@ sendLogQueue gcp@GCPState{..} runSync = do
           void $
           forkFinally send (handleResult)
 
+logException :: Exception e => Lgr.Handle m -> e -> m ()
+logException handle e = Lgr.logJson handle Lgr.Error $ displayException e
```

Comment:
Should this be somewhere shared? With the rest of the log functions?

Reply:
Should it also record the stack trace? Does displayException usually include that?

Reply:
You don’t get a stack trace for where the exception was thrown unless you compile with profiling info. I suppose we could include the location of the call to `logException` but I am not sure that’s particularly useful.
I generally tend to keep these helpers local until I actually need them somewhere else.

Reply:
Fair enough on both counts.